### PR TITLE
Improvements to signat

### PR DESCRIPTION
**Pull request by matthew-mcallister**
Handles some weird cases which just came up. Also now shows OCF accounts for student groups.

### CODE REVIEW

File: staff/acct/signat [hunk 1]
```diff
@@ -15,60 +15,98 @@ from ocflib.ucb.groups import (
...
+        print(red("User not found."))
         return
 
-    show_groups_by_student_signat(int(attrs['calnetUid'][0]))
+    uid = attrs.get('calnetUid', [None])[0]
+    oid = attrs.get('callinkOid', [None])[0]
+
+    if uid and oid:
+        print(yellow("Both CalNet UID and CalLink OID are defined for " +
+            "this account."))
```
Comment (chriskuehl) [inline staff/acct/signat]: nice

File: staff/acct/signat [hunk 2]
```diff
@@ -15,60 +15,98 @@ from ocflib.ucb.groups import (
...
+            "groups.").format(uid))
         return
 
-    name = directory.name_by_calnet_uid(uid)
-    title = "{} ({})".format(name, uid)
-    columns = [(attrs['name'], uid) for (uid, attrs) in groups.items()]
-    headers = ('Group', 'OID')
+    title = '{} ({})'.format(name, uid)
+    columns = [(attrs['name'], ', '.join(attrs['accounts']) or 'n/a', oid) for
+            oid, attrs in groups.items()]
```
Comment (chriskuehl) [inline staff/acct/signat]: nit: this is really weird indentation for a list comprehension

File: staff/acct/signat [hunk 3]
```diff
@@ -162,15 +200,15 @@ def main():
...
         show_signatories_for_group(args.oid)
-    elif args.subparser_name == 'user':
-        show_groups_by_signat_username(args.username)
-    elif args.subparser_name == 'name':
+    elif args.command == 'user':
+        show_by_username(args.username)
+    elif args.command == 'name':
         show_groups_by_signat_name(' '.join(args.name))
-    elif args.subparser_name == 'group':
+    elif args.command == 'group':
```
Comment (chriskuehl) [inline staff/acct/signat]: nice

**Comment by chriskuehl** [comment]
shipit

File: staff/acct/signat (hunk 2 above)
Comment (matthew-mcallister) [inline staff/acct/signat]: Sorry, I just auto-linebreak after 79 characters. Feel free to apply coding standards.

File: staff/acct/signat (hunk 1 above)
Comment (matthew-mcallister) [inline staff/acct/signat]: Try `signat oid 90006`

**Pull request merged**